hive-plugin/src/main/java/com/wipro/ats/bdre/hiveplugin/hook/LineageHook.java

```diff
@@ -53,34 +55,26 @@ public void run(HookContext hookContext) {
 				    String key = (String)(entry.getKey());
 				    if (LineageConstants.processIdString.equalsIgnoreCase(key)) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)